packages/lit-dev-content/site/docs/v3/releases/upgrade.md

```diff
@@ -1,40 +1,63 @@
 ---
-title: Upgrade guide
+title: Upgrade to Lit 3.x guide
 eleventyNavigation:
-  key: Upgrade guide
+  key: Upgrade to Lit 3.x guide
   parent: Releases
   order: 3
 versionLinks:
   v2: releases/upgrade/
 ---
 
-## Overview
+<div class="alert alert-info">
+
+If you are looking to migrate from Lit 1.x to Lit 2.x, see the [Lit 2.0 upgrade guide](/docs/v2/releases/upgrade/).
 
-Lit 3.0 has very few breaking changes from Lit 2.0. Any projects using Lit 2
-without deprecation warnings should not require any changes to upgrade to Lit 3.
+</div>
 
-Lit 3.0:
+## Overview
 
-- Drops support for IE11
-- Is published as ES2021
-- Removes a few deprecated APIs
+Lit 3.0 has very few breaking changes from Lit 2.0:
+
+- IE11 is no longer supported.
+- Lit's npm modules are now published as ES2021.
+- APIs deprecated with the Lit 2.0 release have been removed.
+- SSR hydration support modules have moved to the `@lit-labs/ssr-client` package.
+- Decorator behavior has been unified between TypeScript experimental decorators and standard decorators.
+- Support was removed for Babel decorators version "2018-09"
 
 Lit 3.0 should require no changes to upgrade from Lit 2.0 for the vast majority
```

Comment:
for consistency's sake, i think unifying to "2.x" in the body of the blog seems better.
```suggestion
Lit 3.0 should require no changes to upgrade from Lit 2.x for the vast majority
```

Reply:
Agreed!